docs/index.md

```diff
@@ -40,6 +40,16 @@ You can test this config by creating a `test.tf` and run terraform commands from
 - Build the infrastructure: `terraform apply`
 
 ```hcl
+variable "project_id" {
+  type        = string
+  description = "Your project ID."
+}
+
+variable "dns_zone" {
```

Comment:
How is this variable used?

Reply:
oops, remain of the old "huge" doc. Removed